src/services/WalletService/walletService.ts

```diff
@@ -32,15 +32,14 @@ export const WalletFactory = (walletId: WalletId): WalletService | any => {
           return unlockWeb3(onSuccess)(networks);
         }
       };
+    case WalletId.LEDGER_NANO_S_NEW:
```

Comment:
Sure about this?
It changes the response type to the same as LEDGER_NANO_S

Reply:
Yep, @blurpesec confirmed that it wasn't disabled for a reason and it currently limits testing if you use the new flow to add accounts, since you can't sign.